packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,33 @@
 # <%= project.name %>
 
-[![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+<% if (project.description) { -%>
+> <%= project.description %>
+<% } -%>
+
+This project was generated with [Loopback CLI](https://github.com/strongloop/loopback-next/tree/master/packages/cli) version <%= cliVersion %>.
+
+## First steps
+
+### Installation
+To proceed with the installation of the dependencies in Loopback you must have `nodejs` >= 10 and `npm` installed
+```sh
+npm install
+```
+### Run
+To start the project is as easy as this
+```sh
+npm run start
+```
```

Comment:
When a new app is created, the dependencies are already installed, so no need to do `npm install`.

Also, the user sees:
```
Next steps:

$ cd app
$ npm start
```
when they create the app with the CLI , so I'm wondering if we need this part.

Reply:
I would leave it there because there may be multiple users who have access and do not know how it works

Reply:
Okay, but maybe reword the installation one since the dependencies are already installed and say it's how they can install extra dependencies 👍

Reply:
sorry I do not understand him

Reply:
Perhaps something like:

```
The dependencies are already installed for LoopBack 4 application scaffolded using `lb4` command. If you add other dependencies, please make sure you run `npm i` before running.
```

Just wanna clarify, i think we can use `npm start` for starting the project instead of `npm run start`.

Reply:
Ready. it's looking very good